### DISCUSSION TITLE
Adding a new table from Excel on sharepoint

### DISCUSSION THREAD

**Discussion by Faber81** [TabularEditor/TabularEditor3#1423] [🙋♀️TE3 Q&A]
We have a SharePoint with a huge amount of excels. I need only one. I tried to connect directly to the excel via web connector. Everything works fine in PBI Desktop but in TE3 I get DMTS_DatasourceHasNoCredentialError.  

I also tried to connect via SharePoint connector and filter out all the unecessary files but then I get a "sample file"-"transform_file" thing that again works fine in PBI Desktop but fails in TE3 (can't convert to type function error) 

Is there a tutorial about how to import one excel from SharePoint? In theory it should be quite straightforward but I have already lost a lot of time on this thing. 

Thanks!

**Comment by otykier** [comment]
Unfortunately, we don't have native support for SharePoint / Excel in TE3.

I assume you're seeing the `DMTS_DatasourceHasNoCredentialError` when you're trying to perform an **Update table schema** action in TE3?

Due to the limited support, this would only work if you have **Use Analysis Services for change detection** enabled under **Tools > Preferences > Schema Compare**, and also, you must make sure that the credentials for the data source have been correctly set up in the Power BI Service. That is, on the semantic model settings page, please make sure no errors are shown in the **Data source credentials** section:

![image](https://github.com/user-attachments/assets/56015bc4-a7fd-4c38-a2b0-5495cfd4acce)
